mbed_lstools/lstools_base.py

```diff
@@ -176,6 +176,7 @@ def __init__(self):
         "9012": "SEEED_TINY_BLE",
         "9900": "NRF51_MICROBIT",
         "FFFF": "K20 BOOTLOADER",
+        "OSHW":"Unknown DAPLink Implementation",
```

Comment:
How about just `UNKNOWN_TARGET` instead of `Unknown DAPLink Implementation`? Or `OSHW`?

mbed-ls doesn't list "DAPLink implementations", it uses the DAPLink implementation to ID what target is on the other side of the DAPLink circuit.